auslib/db.py

```diff
@@ -2423,6 +2429,10 @@ def hasRole(self, username, role, transaction=None):
         roles_list = [r['role'] for r in self.getUserRoles(username, transaction)]
         return role in roles_list
 
+    def hasUser(self, username, role, transaction=None):
```

Comment:
"roleHasUser" would be a better name for this, since the class is "Permissions".

Reply:
Cool, will make this change.